app/views/_user_util_links.html.erb

```diff
@@ -34,6 +34,11 @@ Unless required by applicable law or agreed to in writing, software distributed
       <% end %>
     </li>
     <% end %>
+    <% if user_signed_in? %>
```

Comment:
I think this works for now, but we probably should change this to work off an ability:
if current_ability.can? :create, Playlist
Or something like that.